src/main/resources/application.yaml

```diff
@@ -38,6 +38,13 @@ spring:
     password: ${POSTGRES_PASSWORD:dbuserprofile}
     properties:
       charSet: UTF-8
+    hikari:
+      minimumIdle: 2
+      maximumPoolSize: 10
+      idleTimeout: 600000
+      poolName: UPHikariCP
+      maxLifetime: 1800000
+      connectionTimeout: 30000
```

Comment:
maxlifetime seems to be too high

Reply:
other reform teams has kept maxlifetime as 60 secs , as compared to them we have kept 30 secs